tests/performance/MemoryTest.cpp

```diff
@@ -197,76 +196,19 @@ TEST_P(MemoryTest, ReadNPartitionsFromVersionedLayer) {
   });
 }
 
-TEST_P(MemoryTest, PrefetchPartitionsFromVersionedLayer) {
```

Comment:
Is there a reason why you removed this.

Reply:
An old test that we don't use, we have another one is a separate file.

Reply:
Ok.